clients/go/admin/client.go

```diff
@@ -192,6 +195,10 @@ func initializeClients(ctx context.Context, cfg *Config, tokenCache pkce.TokenCa
 		opts = append(opts, authOpt)
 	}
 
+	if cfg.LoadBalancingPolicy != "" {
+		opts = append(opts, grpc.WithDefaultServiceConfig(fmt.Sprintf(`{"loadBalancingPolicy":"%v"}`, cfg.LoadBalancingPolicy)))
```

Comment:
Should we expose the ServiceConfig as the config? instead of the load balancing policy?
Docs are here: https://github.com/grpc/grpc/blob/master/doc/service_config.md
Protobuf type is here: https://github.com/grpc/grpc-proto/blob/master/grpc/service_config/service_config.proto#L625

It looks like there is no current Go type (the existing one is still used by the code but marked as deprecated)... so we can make this config a string with links to the documentation.

wdyt @ckiosidis, @honnix ?

Reply:
Right. That makes sense. Is there any other package we will need to preload?

Reply:
I pushed a change, let me know what you think @EngHabu